### DISCUSSION TITLE
Maya render with arnold error Pyside module not detected

### DISCUSSION THREAD

**Discussion by Sharkblack3D** [BramStoutProductions/MiEx#117] [Q&A]
hi,
I'm getting this error when importing the Python file in Maya 2025. What should I do?
`# Error: ModuleNotFoundError: file <maya console> line 31: No module named 'PySide2'`

**Comment by Sharkblack3D** [comment]
I succeeded but I can't get the texture of the banners, these are neon purple

**Comment by BramStout** [comment]
In Maya 2025 they updated from Qt5 to Qt6 (Qt is what Maya uses for the UI). I'm not on Maya 2025 myself (still on Maya 2024) so I hadn't noticed that yet. In Qt6, it's no longer `PySide2` but `PySide6`. I'm currently busy with client work, but when I'm done with that, I'll update the script to work with Qt6 as well.

Regarding the texture of the banners. Since banners are essentially multiple textures layered on top of each other, MiEx does this layering when exporting and saves it out as a texture in the chunks folder, in a `banners` folder. Could you double check that there are actually banner textures in there? If there are, could you then go into the shading network in Maya for the banner materials and double check that the texture file paths in there are set correctly?
